tests/CMakeLists.txt

```diff
@@ -57,11 +56,17 @@ if (NOT GTestMain_FOUND)
     )
 endif()
 
+# cmake_find_package.yml explicitly tests find_package support used by package managers
 if (CMAKE_CURRENT_SOURCE_DIR STREQUAL CMAKE_SOURCE_DIR)
-    # CMake has been started independently in this directory with tests. Do
-    # import the globally installed Guidelines Support Library and test it
-    # instead of the current version from the include/ folder.
-    find_package(Microsoft.GSL REQUIRED)
+    find_package(Microsoft.GSL CONFIG REQUIRED)
+    enable_testing()
+
+    # Test version number support
+    if (NOT DEFINED Microsoft.GSL_VERSION)
```

Comment:
this is coming from `find_package`, correct?

Reply:
Correct.

The find_package command will create the variable `<PackageName>_VERSION`.
In this case <PackageName> is `Microsoft.GSL`
https://cmake.org/cmake/help/latest/command/find_package.html?highlight=find_package#config-mode-version-selection